.github/workflows/linux_builds.yml

```
@@ -148,7 +148,14 @@ jobs:
         run: |
           sudo rm -rf /usr/share/dotnet/sdk/*
 
-      - name: Setup older .NET SDK as baseline
+      - name: Setup older .NET SDK 6 as baseline
+        if: matrix.build-mono
+        uses: actions/setup-dotnet@v4
+        with:
+          # Targeting the oldest version we want to support to ensure it still builds.
+          dotnet-version: 6.0.100
```

You can also just do this:

```suggestion
          dotnet-version: |
              6.0.100
              8.0.100
```